forc-test/src/execute.rs

```diff
@@ -87,14 +93,27 @@ impl TestExecutor {
                 }));
             output_index += 1;
         }
-        let consensus_params = tx_builder.get_params().clone();
+
+        let mut consensus_params = tx_builder.get_params().clone();
         // Temporarily finalize to calculate `script_gas_limit`
         let tmp_tx = tx_builder.clone().finalize();
         // Get `max_gas` used by everything except the script execution. Add `1` because of rounding.
         let max_gas =
             tmp_tx.max_gas(consensus_params.gas_costs(), consensus_params.fee_params()) + 1;
         // Increase `script_gas_limit` to the maximum allowed value.
-        tx_builder.script_gas_limit(consensus_params.tx_params().max_gas_per_tx - max_gas);
+        tx_builder.script_gas_limit(consensus_params.tx_params().max_gas_per_tx() - max_gas);
+
+        // We need to increase the tx size limit as the default is 110 * 1024 and for big tests
+        // such as std and core this is not enough.
+        // and for big tests such as std, and core this is not enough.
+        let mut tx_params = *consensus_params.tx_params();
+        let script_params = *consensus_params.script_params();
+        let script_params = script_params
+            .with_max_script_length(u64::MAX)
+            .with_max_script_data_length(u64::MAX);
+        tx_params.set_max_size(u64::MAX);
```

Comment:
Aren't these already set in `maxed_consensus_params()`?

Reply:
Nice catch! thanks